UKHO.ExchangeSetService.API/UKHO.ExchangeSetService.Common/Logging/EventIds.cs

```diff
@@ -856,8 +856,31 @@ public enum EventIds
         /// <summary>
         /// 805211 - Download readme.txt file is failed.
         /// </summary>
-        ErrorInDownloadReadMeFile = 805211
-
+        ErrorInDownloadReadMeFile = 805211,
+        /// <summary>
+        /// 805212 - Request for exchange set service product names endpoint is started.
+        /// </summary>
+        ESSPostProductNamesRequestStart = 805212,
+        /// <summary>
+        /// 805213 - Request for exchange set service product names endpoint is completed.
+        /// </summary>
+        ESSPostProductNamesRequestCompleted = 805213,
+        /// <summary>
+        /// 805214 - Request for exchange set service call started.
```

Comment:
```suggestion
        /// 805214 - Request for exchange set service call by product names started.
```
similar change in 805215

Reply:
Changed summary